javascript/ql/src/Statements/UseOfReturnlessFunction.ql

```diff
@@ -0,0 +1,103 @@
+/**
+ * @name Use of returnless function.
```

Comment:
Interesting suggestion to flip the perspective like that.  
I have not looked into where the error is, I'll do that.

Reply:
It seems to me that the mistake is mostly at the callsite.  
Altough it is often a hard call to make whether the mistake is at the callsite or in the function body.

Reply:
Ok. Lets keep the alert at the call site then as that is where you will be hit by the mistake. It is a shame that we have to avoid double-reporting for so many other queries though.